java/HealthVault Mobile/HVSample/src/main/res/values/dimens.xml

```diff
@@ -81,4 +81,50 @@
 	<dimen name="profileimagelayout_marginleftt">142dp</dimen>
 	<dimen name="profileimagelayout_marginbottom">28dp</dimen>
 
+	<!--Medications -->
+	<dimen name="medication_main_marginleft">20dp</dimen>
+	<dimen name="medication_main_marginright">125dp</dimen>
+	<dimen name="medication_main_margintop">22dp</dimen>
+	<dimen name="medication_main_height">24dp</dimen>
+	<dimen name="medication_main_width">215dp</dimen>
+	<dimen name="medication_list_width">360dp</dimen>
+	<dimen name="medication_list_height">88dp</dimen>
+	<dimen name="medication_textsize">24sp</dimen>
+	<!--Medications record-->
+	<dimen name="medicationname_marginleft">20dp</dimen>
+	<dimen name="medicationname_margintop">12dp</dimen>
+	<dimen name="medicationname_width">86dp</dimen>
+	<dimen name="medicationname_height">24dp</dimen>
+	<dimen name="medicationname_textsize">18sp</dimen>
+
+	<dimen name="medicationdose_marginleft">20dp</dimen>
+	<dimen name="medicationdose_width">200dp</dimen>
+	<dimen name="medicationdose_height">20dp</dimen>
+	<dimen name="medication_dose_textsize">15sp</dimen>
+
+	<dimen name="medicationprescribe_marginleft">20dp</dimen>
```

Comment:
As mentioned before, we should use generic names here and link common values. For example, we create multiple margin properties in this file with the value of 20dp. Instead of making a 20dp margin for each page, make one generic margin with the value of 20dp and use it in multiple places. This is how we used to do the older apps. We had generic sizes like `@dimen/smallSpace` or `@dimen/mediumSpace`